voice/bxml/callbacks/dtmf.md

```diff
@@ -0,0 +1,55 @@
+{% method %}
+## DTMF Event
+
+The dtmf event is sent for every digit detected after a [`<StartGather>`](../verbs/startGather.md) verb is executed.
+
+### Expected response
+
+```http
+HTTP/1.1 204
+```
+
+### Properties
+| Property         | Description |
+|:-----------------|:------------|
+| eventType        | The event type, value is `dtmf`. |
+| accountId        | The user account associated with the call. |
+| applicationId    | The id of the application associated with the call. |
+| to               | The phone number that received the call, in E.164 format (e.g. +15555555555). |
+| from             | The phone number that made the call, in E.164 format (e.g. +15555555555). |
+| direction        | The direction of the call. Either `inbound` or `outbound`. The direction of a call never changes. |
+| callId           | The call id associated with the event. |
+| callUrl          | The URL of the call associated with the event. |
+| startTime        | Time the call was started, in ISO 8601 format. |
+| answerTime       | Time the call was answered, in ISO 8601 format. |
+| parentCallId     | (optional) If the event is related to the B leg of a `<Transfer>`, the call id of the original call leg that executed the `<Transfer>`. Otherwise, null. |
+| transferCallerId | (optional) If the event is related to the B leg of a `<Transfer>`, the phone number used as the `from` field of the B-leg call, in E.164 format (e.g. +15555555555). Otherwise, null. |
+| transferTo       | (optional) If the event is related to the B leg of a `<Transfer>`, the phone number used as the `to` field of the B-leg call in E.164 format (e.g. +15555555555). Otherwise, null. |
+| tag              | (optional) The `tag` specified earlier in the call. If no `tag` was specified or it was previously cleared, null. |
+| digit            | The digit collected in the call. |
```

Comment:
Move nearer to the top since it's the most important part?

Reply:
Moved right after `callId`.